Lang/Spanish.pj.Lang

```diff
@@ -479,20 +479,20 @@
 #2038#  "Cargada"
 #2039#  "Cargando imagen"
 #2040#  "No se puede abrir un ROM porque los plugins no se han inicializado correctamente."
-#2041#  "Estás seguro de que realmente quieres borrar esto?"
+#2041#  "¿Estás seguro de que realmente quieres borrar esto?"
 #2042#  "Borrar Truco"
 #2043#  "El nombre del truco ya está en uso."
 #2044#  "Haz alcanzado la cantidad máxima de trucos para este ROM."
 #2045#  "Inicializando el plugin"
-#2046#  "No has seleccionado una tecla virtual para asignarla al elemento del menú."
+#2046#  "No has seleccionado una tecla virtual para asignar al elemento del menú."
 #2047#  "Necesitas seleccionar un elemento del menú al cual asignarle esta tecla."
 #2048#  "El acceso directo ya está asignado a otro elemento del menú."
 #2049#  "Ningún acceso directo ha sido seleccionado para ser removido."
-#2050#  "ROM cargada. Esperando por la emulación para iniciar."
-#2051#  "Project64 beta es solo para miembros.\n\nSi tienes una cuenta en pj64.net, no deberías estar viendo este error!!\nPor favor contacta con nosotros en el sitio."
+#2050#  "ROM cargado. Esperando el inicio de la emulación."
+#2051#  "Project64 Beta es sólo para miembros.\n\n¡¡Si tienes una cuenta en pj64-emu.com, no deberías estar viendo este error!!\nPor favor contacta con nosotros en el sitio."

```

Comment:
The above here caused @MELERIX to do this change:
https://github.com/MELERIX/project64/commit/a4f8f52de9eeb8c170983dbec56217bcb6c60869#diff-f9de25fe9c46bd6e519b637c46b66814L492

At least in English, that I know is not accurate because "beta" is a common noun, not a proper noun.  There's no reason to capitalize it to "Project64 Beta" because the name of the project isn't "Project64 Beta"; it's "Project64".  Also, beta versions aren't unique to just Project64; they are a common thing.

It's like capitalizing "dollars", "rupees", "fall", "summer", "winter" ... you're never supposed to capitalize those things unless the format or something warrants it.